server/server.go

```diff
@@ -288,25 +288,9 @@ func (db passwordDB) Login(ctx context.Context, s connector.Scopes, email, passw
 		}
 		return connector.Identity{}, false, nil
 	}
-
-	// Return an error if password-hash comparison takes longer than 10 seconds
-	errCh := make(chan error, 1)
-	go func() {
-		errCh <- bcrypt.CompareHashAndPassword(p.Hash, []byte(password))
-	}()
-	select {
-	case err = <-errCh:
-		if err != nil {
-			return connector.Identity{}, false, nil
-		}
-	case <-time.After(time.Second * 10):
-		var cost int
-		if cost, err = bcrypt.Cost(p.Hash); err == nil {
-			err = fmt.Errorf("password-hash comparison timeout: your bcrypt cost = %d, recommended cost = %d", cost, recCost)
-		}
-		return connector.Identity{}, false, err
+	if err := bcrypt.CompareHashAndPassword(p.Hash, []byte(password)); err != nil {
```

Comment:
We probably want to do the checkCost check here too. If a user provides a static password with a large bcrypt cost, we cant to error out early.